src/components/common/ChainIndicator/index.tsx

```diff
@@ -26,6 +26,8 @@ const ChainIndicator = ({ chainId, className, inline = false }: ChainIndicatorPr
     }
   }, [chainConfig])
 
+  if (!chainConfig?.chainName) return null
```

Comment:
The change here is needed for the situation where an app is supported on a chain we don't support to prevent spending computational resources on filtering. Currently, if you pass an unsupported chain id to this component, it renders a space